meraki/rest_session.py

```diff
@@ -6,12 +6,52 @@
 from datetime import datetime
 
 import requests
+from requests.utils import to_key_val_list
+from requests.compat import basestring, urlencode
 
 from meraki.__init__ import __version__
 from meraki.common import *
 from meraki.config import *
 
 
+def encode_params(_, data):
+    """Encode parameters in a piece of data.
```

Comment:
I'd like to request more documentation here explaining why we're doing this. You can keep it high-level, but it should make sense to both folks familiar with the `requests` library, and also to folks who are not Python experts.

You might could reference issues in the GitHub tracker as well.